samza-core/src/main/scala/org/apache/samza/coordinator/server/JobServlet.scala

```diff
@@ -22,19 +22,22 @@ package org.apache.samza.coordinator.server
 
 import java.util.concurrent.atomic.AtomicReference
 
-import org.apache.samza.SamzaException
-import org.apache.samza.job.model.JobModel
-import org.apache.samza.util.Logging
+import javax.servlet.http.{HttpServlet, HttpServletRequest, HttpServletResponse}
 
 /**
- * A servlet that dumps the job model for a Samza job.
+ * Serves the JSON serialized job model for the job.
  */
-class JobServlet(jobModelRef: AtomicReference[JobModel]) extends ServletBase with Logging {
-  protected def getObjectToWrite() = {
+class JobServlet(jobModelRef: AtomicReference[Array[Byte]]) extends HttpServlet {
+  override protected def doGet(request: HttpServletRequest, response: HttpServletResponse) {
```

Comment:
Should we update the 
doGet implementation in ServletBase ?

Reply:
Good catch. Meant to delete `ServletBase`, it's no longer used.